### PR TITLE
notes for beta release

### PR DESCRIPTION
**Pull request by LianaHus**

### CODE REVIEW

File: release-process.md [hunk 1]
```diff
@@ -24,6 +24,26 @@ This document includes:
...
+ - git checkout origin/master
+ - git tag v(version-number) (with "vx.x.x-beta.1")
+ - git push --tags
+ - github-changes -o ethereum -r remix-ide -a --only-pulls --use-commit-body --only-merges --between-tags previous_version...next_version
+ - publish a beta release in github using the changelog
+ - rm -rf node_modules
+ - npm install
+ - remove all soljson.js files in root folder
+ - npm run build
+ - npm publish
```
Comment (LianaHus) [inline release-process.md]: should i publish it?

File: release-process.md [hunk 2]
```diff
@@ -24,6 +24,26 @@ This document includes:
...
+ - drop zip file to the beta release
```
Comment (LianaHus) [inline release-process.md]: from where?

Comment (LianaHus) [reply release-process.md]: https://github.com/ethereum/remix-live-alpha

File: release-process.md (hunk 1 above)
Comment (LianaHus) [reply release-process.md]: no

File: release-process.md (hunk 2 above)
Comment (yann300) [reply release-process.md]: the place where to get the zip file should be put in there

**Pull request merged**